Sources/NIO/SelectableEventLoop.swift

```diff
@@ -555,6 +561,11 @@ internal final class SelectableEventLoop: EventLoop {
             }
         }
     }
+    
+    func withControlMessageBytes<ReturnType>(
+        _ body: (UnsafeMutableRawBufferPointer) throws -> ReturnType) rethrows -> ReturnType {
+        return try body(self.controlMessageBuffers)
+    }
```

Comment:
How useful is this abstraction method? It doesn't seem to add any clarity at all.